pyspeckit/spectrum/models/ammonia.py

```diff
@@ -162,7 +164,7 @@ def ammonia(xarr, tkin=20, tex=None, ntot=14, width=1, xoff_v=0.0,
         these equations, not just the final one)
         """
         dT0 = 41.5                    # Energy diff between (2,2) and (1,1) in K
-        trot = tkin/(1+tkin/dT0*np.log(1+0.6*np.exp(-15.7/tkin)))
+        trot = trot/(1+trot/dT0*np.log(1+0.6*np.exp(-15.7/trot)))
```

Comment:
My mistake, I had done a global search-and-replace on tkin->trot.  The latest version undoes this and makes more clear when and where tkin, trot are to be used